test/irmin-git/jbuild

```diff
@@ -3,7 +3,7 @@
 (library
  ((name      test_git)
   (modules   (test_git))
-  (libraries (irmin-test irmin-git git-unix))))
+  (libraries (irmin-test checkseum.c irmin-git git-unix))))
```

Comment:
this should not be needed, e.g. `checkseum` the concrete implementation (C or OCaml) should be chosen at link time only.

Reply:
Without this the `irmin-git` tests wouldn’t build though. What is the preferred way of handling this?

Reply:
You should add `checkseum.c` to all built executable, e.g. the test binaries too.

Reply:
Got it, I've moved the linking to `checkseum.c` into the executable sections of the necessary jbuild files. Everything seems to be building correctly now (including tests and examples) using the most recent commit.